src/kaocha/version_check.clj

```diff
@@ -0,0 +1,14 @@
+(ns kaocha.version-check
+  (:require  [kaocha.output :as output]
+            [slingshot.slingshot :refer [throw+]]))
+
+(defn check-version-minimum 
+  "Checks that Clojure has at least a minimum version"
+  [major minor]
+  (when-not (or (and (= (:major *clojure-version*) major) (>= (:minor *clojure-version*) minor))
+                (>= (:major *clojure-version*) (inc major) ))
+    (let [msg (format "Kaocha requires Clojure %d.%d or later." major minor)]
+      (output/error msg)
+      (throw+ {:kaocha/early-exit 251} msg))))
+
+(check-version-minimum 1 9)
```

Comment:
Maybe wrap this in a `(comment)`

Reply:
This is actually needed for the check to happen upon importing `kaocha.version-check`. I could change each file to specifically call this function, but since 1.9 is needed for Kaocha as a whole, I'm not sure it's worth it.

Reply:
Of course 🤦 never mind me!